lib/app_store_connect/object/included.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'active_support/concern'
+
+module AppStoreConnect
+  module Object
+    module Included
+      extend ActiveSupport::Concern
+
+      included do
+        attr_reader :included
+
+        klass = Class.new do |i|
+          i.send(:define_method, :initialize) do |objects|
+            instance_variable_set('@objects', [*objects])
+          end
+
+          def to_a
+            @objects
+          end
+        end
+
+        const_set('Included', klass)
```

Comment:
Is this only so that the class including this module can be `initialized` with `included`? 🤔

Reply:
@kyledecot why don't just add `included` as a class field instead of having it as a separate module? What benefits do you see in your current approach?

Reply:
Good question–I usually do this to keep classes/modules as simple as possible to follow the single responsibility principle. In this particular case it probably doesn't make a lot of difference however most of the codebase follows this convention so it's also done for consistency.